### PR TITLE
feat: replace delegate status icons

### LINKED ISSUE #693: Accessibility: Status images in the delegate table are not keyboard accessible
**Expected Behavior**

A screen reader, or even a sighted user who is only using a keyboard, should be able to understand the "status" column in the main Delegate table here:

https://dexplorer.ark.io/delegate-monitor

**Current Behavior**

The current popover for Status seems to be an mouse-only 'on-hover' event, which is not accessible for keyboard-only users, including the blind and those who are unable to use a mouse. The image itself does not contain any alt text, and a screen reader simply says "image" when navigated to, and the popover is not triggered.

**Possible Solution**

Please either add some alt text that mimics what's in the popover, or trigger the popover on-focus on that image in the table so the screen reader can announce it. Example, include's aria-role and aria-describedBy attributes:
https://heydonworks.com/practical_aria_examples/#input-tooltip

### LINKED ISSUE #695: Accessibility: Status images in the delegate table are not suitable for users with impaired acuity
**Expected Behaviour**

Users should be **easily** able to tell the difference between the three statuses.

**Current Behaviour**

It is not possible for certain colourblind people with additional acuity problems to tell the difference. Try crossing your eyes and looking at the explorer, imagining there is no colour.

**Possible Solution**

1. Use different larger shapes. A full sized green square, red circle or yellow triangle for the respective statuses would provide more definition to the icons rather than dots and dashes. Different shapes would be more easily differentiated for users with acuity problems.

Alternatively:

2. Just use ✅, ❌ and ⚠️ unicode emoji which will render natively and already convey enough visual information to be understandable without trying to work out "what do two dots, a dot and a dash or two dashes mean" as they are already intuitive in context. Bonus points for the fact that they work natively with screen readers re: #693. No need to reinvent the wheel.

### PR DESCRIPTION
**Pull request by dated**
- [x] replaces the delegate status icons (closes #695)
- [x] improves accessibility of the delegate monitor (closes #693)

image

## What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [x] Refactor
- [ ] Performance
- [ ] Tests
- [ ] Build
- [ ] Documentation
- [ ] Code style update
- [ ] Continuous Integration
- [ ] Other, please describe:

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Does this PR release a new version?

- [ ] Yes
- [x] No

If yes, please describe the impact and migration path for existing applications:

**The PR fulfills these requirements:**

- [x] It's submitted to the `develop` branch, _not_ the `master` branch
- [x] All tests are passing
- [ ] New/updated tests are included

If adding a **new feature**, the PR's description includes:

- [x] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

### CODE REVIEW

**Comment by alessiodf** [comment]
Are these the same design that @ItsANameToo showed us a few weeks ago?

**Comment by ItsANameToo** [comment]
Yes @alessiodf

**Comment by alessiodf** [comment]
This may be out of scope but is it not worthwhile having a separate icon (like, for example, a blue circle) to denote the _never forged_ delegates rather than showing the not forging symbol?

**Review by ItsANameToo — DISMISSED** [review]

**Comment by ghost** [comment]
A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Review by ItsANameToo — APPROVED** [review]

**Comment by ghost** [comment]
A contributor has approved this PR. A maintainer will merge this PR shortly. If it shouldn't be merged yet, please leave a comment saying so and we'll wait.

_Thank you for your contribution!_

**Comment by ItsANameToo** [comment]
conflict :trollface:

**Comment by ghost** [comment]
Your pull request has been merged and marked as tier 4. _It will earn you $20 USD._

File: app/Http/Livewire/PriceStats.php
Comment (alfonsobries) [inline app/Http/Livewire/PriceStats.php:53]: These values are for creating the chart that is used as a placeholder:

image

**Comment by samharperpittam** [comment]
@alfonsobries 

Have some questions on this PR.

1. With the `.env` set to `EXPLORER_NETWORK=development`, the Explorer is then using our devnet. To which, as DARK tokens do not have any value, the price stat should probably use the placeholder? 

2. With the `.env` set to `EXPLORER_NETWORK=production`, the Price stat is showing a price of $7.38 which is incorrect. Is it pulling in the price from cryptocompare now? 

3. When viewing stats which are negative, the percentage value doesn't appear to be aligned with the price value as shown in the image below (probably only a few pixels out). 
image

4. It appears to me that the stats are showing the refresh animation every 8 seconds rather than every 60?

**Comment by alfonsobries** [comment]
@samharperpittam 

1. The placeholder is shown according to the value in the `explorer.development.canBeExchanged`(`config/explorer.php`) setting. 

If `EXPLORER_NETWORK_CAN_BE_EXCHANGED` is set to `development`. The value of that setting is `false` by default so you should see the placeholder. If you are seeing something different you probably have the value of`EXPLORER_NETWORK_CAN_BE_EXCHANGED` set to `true`, simply remove that setting from the `env` file (or set it to `false`).

image


2. that value comes from cryptocompare maybe you need to remove the cache: `php artisan cache:clear` (I see 1.45)

3. and 4. working on it

**Comment by alfonsobries** [comment]
@samharperpittam just fixed points 3 and 4, so just waiting for your comment on the rest of the feedback

**Comment by samharperpittam** [comment]
Thanks for the info @alfonsobries  - Much appreciated!

Everything is working as expected now.

My only other comment and it's pretty minor, is that when compared with the design, the gaps between each component in the grid feels a bit large.

**Design**
image

**Implementation**
image

If we can bring that a little closer to the designs then I'm happy with this 👍

**Comment by alfonsobries** [comment]
@samharperpittam ✅

File: resources/lang/en/general.php
Comment (ItsANameToo) [inline resources/lang/en/general.php]: what's this for? 🤔

File: resources/views/livewire/price-stats.blade.php
Comment (ItsANameToo) [inline resources/views/livewire/price-stats.blade.php]: can this be moved to a js file? would make it easier to maintain than having it in the php file

File: resources/views/livewire/network-status-block.blade.php
Comment (ItsANameToo) [inline resources/views/livewire/network-status-block.blade.php]: can this be changed to poll every 8 seconds (block time) so height/supply don't go out of sync for a minute while new blocks are being generated? Preferably the polling timeframe here should be taken from a configuration file so it's easy to adjust if needed (e.g. in case of an ark fork having a different block time). Not sure how well livewire works with that

Comment (alfonsobries) [reply resources/views/livewire/network-status-block.blade.php]: @ItsANameToo take it the from `Network::blockTime()`

**Comment by alfonsobries** [comment]
@ItsANameToo should be ready now. I wasn't able to replicate the issue you had with the chart out of bonds but refactored the chart to have a fixed size and ensured to set the min/max value of the chart, can you test it again?

**Pull request merged**
